### DISCUSSION TITLE
プレイヤーがセーブポリシーを確認できるようにする

### DISCUSSION THREAD

**Discussion by matsuyuki-a** [WWAWing/WWAWing#554] [Ideas]
- ヘルプ表示で確認できるようにする
- ロード中にポリシーを確認できるようにする

**Comment by enjoyablespage** [comment]
セーブポリシーというのは、wwawingがセーブデータをどのように扱いますよ、というものですか？

**Comment by matsuyuki-a** [comment]
はい、セーブデータの取り扱い方についてのことです。
確かにそのあたりの説明が抜けていたので補足させていただきます。

現在、WWAWingにはセーブデータの取り扱いが2種類あり、

- ワールド名・暗証番号の変更 もしくは マップデータの内容が変更された場合にセーブデータが無効になる (Java版相当)
- ワールド名・暗証番号の変更 でのみセーブデータが無効になる (Wing ではこちらがデフォルト)

から、WWA設置者が選ぶことができます 。この設定は [data-wwa-disallow-load-old-save 属性 ](https://wwawing.notion.site/data-wwa-disallow-load-old-save-daf060e8991d40f29d36a02049327d29) で変更することができます。

ですが、現状ではプレイヤーがどちらの設定になっているかを簡単に知ることができないので、ヘルプ表示やゲーム開始時などに確認できないかなと考えているところです。
